app/routes/user/endorsements.tsx

```diff
@@ -215,49 +222,101 @@ export function EndorsementRequestCard({
 }) {
   const fetcher = useFetcher()
   const disabled = fetcher.state !== 'idle'
+  const [showMore, setShowMore] = useState(false)
+  const [isOverflowing, setIsOverflowing] = useState(true)
+  const ref = useRef<HTMLDivElement>(null)
+
+  function overflowing() {
+    if (ref.current)
+      setIsOverflowing(ref.current?.offsetWidth < ref.current?.scrollWidth)
+  }
+
+  useEffect(() => overflowing(), [])
 
   return (
     <fetcher.Form method="POST">
       {role === 'endorser' ? (
         <Grid row style={disabled ? { opacity: '50%' } : undefined}>
-          <div className="tablet:grid-col flex-fill">
+          <div className="tablet:grid-col flex-fill order-first">
             <div className="margin-y-0">
-              <strong>{endorsementRequest.requestorEmail}</strong>
+              <strong className="">{endorsementRequest.requestorEmail}</strong>
             </div>
+            {endorsementRequest.note && (
+              <div className="margin-top-0 margin-bottom-1">
+                Comments from the user:
+                {isOverflowing && (
+                  <Button
+                    type="button"
+                    unstyled
+                    aria-label="Show or hide the comments provided by the requestor"
+                    aria-expanded={showMore}
+                    onClick={() => setShowMore(!showMore)}
+                  >
+                    Show{' '}
+                    {showMore ? (
+                      <>
+                        less <Icon.ArrowDropUp />
+                      </>
+                    ) : (
+                      <>
+                        more <Icon.ArrowDropDown />
+                      </>
+                    )}
+                  </Button>
+                )}
+              </div>
+            )}
+          </div>
+          <div className="tablet:grid-col flex-auto order-last">
+            <ButtonGroup type="default">
+              <input
+                type="hidden"
+                name="requestorSub"
+                value={endorsementRequest.requestorSub}
+              />
+              <Button
+                type="submit"
+                disabled={disabled}
+                name="intent"
+                value="approved"
+              >
+                Approve
+              </Button>
+              <Button
+                type="submit"
+                outline
+                disabled={disabled}
+                name="intent"
+                value="rejected"
+                className="margin-y-1 tablet:margin-y-0"
```

Comment:
What do the manual margin settings do here?

Reply:
Nothing anymore! removing these too